src/test/kotlin/org/rustSlowTests/RsHighlightingPerformanceTest.kt

```diff
@@ -89,14 +90,24 @@ class RsHighlightingPerformanceTest : RsWithToolchainTestBase() {
                 }
             }
 
+        val caretsToAdd = mutableListOf<VisualPosition>()
+
         myFixture.file.descendantsOfType<RsFunction>()
             .asSequence()
             .mapNotNull { it.block?.stmtList?.lastOrNull() }
             .forEach { stmt ->
                 myFixture.editor.caretModel.moveToOffset(stmt.textOffset)
-                myFixture.type("a<caret>;")
+                myFixture.type("aaa;")
```

Comment:
Let's replace "aaa;" with "Hash;" to know exactly what variants will be processed.

And also, we can test "a;" - for this you need to increase the limit of completion variants. This can be done inside `repeatTest` with this function:
```kotlin
private fun setCompletionVariantLimit(value: Int) =
    Registry.get("ide.completion.variant.limit").setValue(value, testRootDisposable)
```